patch/ros-jazzy-ur-client-library.win.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 2368174..b265710 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -101,6 +101,8 @@ install(TARGETS urcl EXPORT urcl_targets
+ )
+ install(DIRECTORY include/ DESTINATION include)
+ 
++install(FILES 3rdparty/endian/endian.h DESTINATION include)
```

Comment:
This may be prone to collision with other packages, can't we install that header in the ur client folder as done in humble:  https://github.com/RoboStack/ros-humble/blob/main/patch/ros-humble-ur-client-library.win.patch

Reply:
addressed at https://github.com/RoboStack/ros-jazzy/pull/51/commits/adb81ff6599ae4a62079154ffeb170e7f45a9061